README.md

```diff
@@ -1,15 +1,17 @@
-# SIG: Web - website2017 - an amazing project
-This is [CompSoc Edinburgh](https://comp-soc.com)'s website, built by SIG: Web.
+# SIG: Web - website2020 (up to date!) - a super duper amazing project
```

Comment:
actually shit i didn't realize this was actually very out of date

Reply:
Can we make this just "website" as opposed to including 2020? is there a need for the year?

Reply:
I think it's important to memorialise0 the great year that is 2020